README.md

```diff
@@ -45,7 +45,79 @@ Host your own `amazon-connect-chat-interface.js` bundle file and provide the lin
 
 ![Host Widget Snippet Integrated Custom UI](/.github/screenshots/custom-chat-widget-interface-screenshot.png)
 
-### Option 3: Customized Widget and Chat Interface UI (Self-Hosted)
+### Option 3: Customize Widget Start Chat Response
+
+Fully customize the chat widget's Start Chat response and API/computing logic. You can call your own backend to start the chat contact instead of the Amazon-hosted backend. This allows you to have more control over the start chat experience.
+
+Follow the instructions in the [startChatContactAPI](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/cloudformationTemplates/startChatContactAPI) section of the Git repository README to create an API to call the StartChatContact API. Once the API is created, you can call it in the CustomStartChat callback function and send the response back in the widget script.
```

Comment:
Looks good overall.  Only note is that it almost sounds like we only support using this method to spin up a back-end.  Maybe we can prefix this paragraph with... "To get help setting up your own start chat backend,"  or "For additional instructions on setting up your own start chat backend, follow ...".